airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -728,7 +728,8 @@ def create_admin_standalone(self) -> tuple[str | None, str | None]:
         if not user_exists:
             print(f"FlaskAppBuilder Authentication Manager: Creating {user_name} user")
             role = self.find_role("Admin")
-            assert role is not None
+            if TYPE_CHECKING:
+                assert role is not None
```

Comment:
I'm not 100% sure that this check uses for only make `mypy` happier

@vincbeck WDYT is it possible that role Admin not existed? And we probably should raise actual error here, e.g. `LookupError` or something similar

Reply:
I guess it is totally possible that the role does not exist (if the user removes for instance). I would just throw `AirflowException` in that case

Reply:
We had a discussion about it and the `if TYPE_CHECKING: assert` was the only one that we decided to "Allow". But I am also for completely getting rid of it especially if we can add ruff rule to take care about it. There is very little benefit of using assert instead of something `if-y` here (like raising exception expliditly here is  "good enough" for me if the trade-off is we will get "don't use assert" automated and future proof.

Reply:
@vincbeck Ok, I will change the code to raise an error

@potiuk I've forgot to attach ruff check without fixes in code, update PR description